### DISCUSSION TITLE
May I use tests of this project for `swc-ecosystem-ci`?

### DISCUSSION THREAD

**Discussion by kdy1** [umijs/dumi#1948] [General]
Hi. I'm the author of SWC.

I looked at various ways to make swc more stable, and as a result of the effort, I created swc ecosystem CI. https://github.com/swc-project/swc-ecosystem-ci

It runs tests from projects using @swc/core with the new version of @swc/core to verify that the new version of @swc/core does not introduce a regression.

May I utilize the tests you wrote? It's basically cloning your repository and running tests in it with the nightly version of @swc/core.

Thank you for using swc, and I look forward to your response!

**Comment by PeachScript** [comment]
Sorry for the late reply, I'm glad this project may helps SWC more stable, and thank you for creating the SWC project!
